_data/projects.yml

```diff
@@ -180,7 +180,7 @@
 'xcore':                        {'lead': ''}
 'xoutputfilter':                {'lead': 'aeberhard'}
 'ycom_board':                   {'lead': 'alxndr-w', 'transferred_at': '2024-10-15'}
-'ycom_impersonate':             {'lead': 'skerbis'}
```

Comment:
Nur als Info: Im Core verwenden wir die Begriffe `impersonate` und `depersonate`
https://github.com/redaxo/redaxo/blob/main/redaxo/src/core/lib/login/api_user_impersonate.php

Reply:
jupp. Das AddOn ist eh archiviert. Aber es stand noch der alte Name in der Liste. Da es kein echtes Impersonate war wurde es in ycom_login_as umbenannt. Also diese Änderung nur der Form wegen. Es ist auch nicht mehr im Installer.